### PR TITLE
adds preternises healing from oil to android species

### PR DESCRIPTION
**Pull request by boodaliboo**
if it heals preternis why shouldnt it heal androids
#### Changelog

:cl:  
rscadd: Added the ability for android species to heal from oil
/:cl:

### CODE REVIEW

**Comment by boodaliboo** [comment]
also i really like the species chem handling code might use it for something else in the future

**Comment by athena413** [comment]
preternis also heal from welding fuel too, but its half as effective as oil and they also arent immune to the toxin damage caused from drinking the welding fuel

good pr tho

**Comment by boodaliboo** [comment]
> preternis also heal from welding fuel too, but its half as effective as oil and they also arent immune to the toxin damage caused from drinking the welding fuel
> 
> good pr tho

oh shit sorry ill make a seperate pr for that after this is merged

File: code/modules/mob/living/carbon/human/species_types/android.dm [hunk 1]
```diff
@@ -23,3 +23,9 @@
 	for(var/X in C.bodyparts)
 		var/obj/item/bodypart/O = X
 		O.change_bodypart_status(BODYPART_ORGANIC,FALSE, TRUE)
+
+/datum/species/android/handle_chemicals(datum/reagent/chem, mob/living/carbon/human/H)
+	. = ..()
+
+	if(H.reagents.has_reagent(/datum/reagent/oil))
+		H.adjustFireLoss(-2*REAGENTS_EFFECT_MULTIPLIER,FALSE,FALSE, BODYPART_ANY)
```
Comment (JamieD1) [inline code/modules/mob/living/carbon/human/species_types/android.dm:31]:
```suggestion
		H.adjustFireLoss(-2 * REAGENTS_EFFECT_MULTIPLIER)
```
Where did you copy paste this from?

File: code/modules/mob/living/carbon/human/species_types/android.dm [hunk 2]
```diff
@@ -23,3 +23,9 @@
 	for(var/X in C.bodyparts)
 		var/obj/item/bodypart/O = X
 		O.change_bodypart_status(BODYPART_ORGANIC,FALSE, TRUE)
+
+/datum/species/android/handle_chemicals(datum/reagent/chem, mob/living/carbon/human/H)
+	. = ..()
+
```
Comment (JamieD1) [inline code/modules/mob/living/carbon/human/species_types/android.dm:29]:
```suggestion
	. = ..()
```

File: code/modules/mob/living/carbon/human/species_types/android.dm [hunk 3]
```diff
@@ -23,3 +23,9 @@
...
+	if(H.reagents.has_reagent(/datum/reagent/oil))
```
Comment (JamieD1) [inline code/modules/mob/living/carbon/human/species_types/android.dm]:
```suggestion
	. = ..()
	if(H.reagents.has_reagent(/datum/reagent/oil))
```

**Review by JamieD1 — APPROVED** [review]

**Pull request merged**